modules/product/config/install/views.view.commerce_products.yml

```diff
@@ -381,17 +382,6 @@ display:
       display_extenders: {  }
     cache_metadata:
       contexts:
-        - languages
-      cacheable: false
-  page_1:
-    display_plugin: page
-    id: page_1
-    display_title: Page
-    position: 1
-    display_options:
-      display_extenders: {  }
-      path: admin/commerce/products
```

Comment:
Not sure what happened, fixed.

Also updated the other views, and removed UUID.